db/sql/dbsetup.sql

```diff
@@ -297,14 +301,15 @@ FROM endpoint_organization AS links
 LEFT JOIN fhir_endpoints AS endpts ON links.url = endpts.url
 LEFT JOIN fhir_endpoints_info AS endpts_info ON endpts.url = endpts_info.url
 LEFT JOIN vendors ON endpts_info.vendor_id = vendors.id
-LEFT JOIN npi_organizations AS orgs ON links.organization_npi_id = orgs.npi_id;
+LEFT JOIN npi_organizations AS orgs ON links.organization_npi_id = orgs.npi_id
```

Comment:
I was trying to find where this was referenced just to make sure using the default made sense to me, but I didn't see it referenced anywhere? Is this actually used?

Reply:
Yeah I don't see it either actually. I just assumed for now only the default mapping should be used, but I also do not see this function referenced anywhere so maybe it is not even needed?

Reply:
I think using the default mapping by default (lol) makes sense, so i'd say we'd either want to keep it as is or remove it!

@mayerm94 do you know why this view exists?

Reply:
Interesting -- I think what happened is we moved from using org_mapping to the endpoint_export view for location information. I'd be okay with us removing this view outright.